compiler/modules/front/stmt.bal

```diff
@@ -225,6 +225,11 @@ function codeGenFunction(ModuleSymbols mod, s:FunctionDefn defn, bir:FunctionSig
         bir:ParamRegister reg = cx.createParamRegister(signature.paramTypes[i], param.name, param.namePos);
         bindings = { name: <string>param.name, reg, prev: bindings, isFinal: true };
     }
+    s:FunctionParam? restParam = defn.restParam;
+    if restParam != () {
+        bir:ParamRegister reg = cx.createParamRegister(<t:SemType>signature.restParamType, restParam.name, restParam.namePos);
```

Comment:
This has duplicated code with non-rest params.

Reply:
fixed (no longer required)